helmctlr/engine_test.go

```diff
@@ -0,0 +1,61 @@
+package helmctlr
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/helm/pkg/chartutil"
+	"k8s.io/helm/pkg/proto/hapi/chart"
+)
+
+type mockEngine struct {
+	out map[string]string
+}
+
+func (e *mockEngine) Render(chrt *chart.Chart, v chartutil.Values) (map[string]string, error) {
+	return e.out, nil
+}
+
+func TestOwnerRefEngine(t *testing.T) {
+	ownerRefs := []metav1.OwnerReference{
+		{
+			APIVersion: "v1",
+			Kind:       "Test",
+			Name:       "test",
+			UID:        "123",
+		},
+	}
+
+	baseOut := `apiVersion: stable.nicolerenee.io/v1
```

Comment:
This string seems right, but ugly. I'm not sure of a more preferred way to do this, but it feels like something other way of doing the string might be less eye strain.